### DISCUSSION TITLE
can we install tailwind css with dok on hugo ?

### DISCUSSION THREAD

**Discussion by wilonweb** [thuliteio/doks#1194] [General]
Hello 
can we install tailwind css with dok on hugo ? 
if yes we to install it ?

**Comment by h-enk** [comment]
Theoretically, it should be possible — see [Hyas Tailwind CSS](https://github.com/gethyas/tailwindcss) for how to install

However, I strongly advise against using both Bootstrap and Tailwind CSS — mainly for possible (processing) conflicts.

**Reply by h-enk** [reply]
See also [Hugo Basic Starter for TailwindCSS v3.x](https://github.com/bep/hugo-starter-tailwind-basic) for an updated setup
